internal/pkg/template/input/auxiliary.go

```diff
@@ -0,0 +1,64 @@
+package input
+
+import (
+	"fmt"
+)
+
+// StepsGroupsExt and nested structs are auxiliary structs with extended information about group/step.
+type StepsGroupsExt []*StepsGroupExt
+
+type StepsExt []*StepExt
+
+type StepsGroupExt struct {
+	StepsGroup `validate:"dive"`
+	Steps      StepsExt
```

Comment:
Bola to celkom vyzva implementovat, a nespravit z toho :spaghetti: 

Doiteroval som to k tomu, ze potrebujem nejake pomocne struktury, aby som nemal milion `map`s ...

Tak som vytvoril tento subor + testy.

Inak vies, ze na Windows je `aux.go` zakazany nazov, a Git ti ho nestiahne?
https://confluence.atlassian.com/bitbucketserverkb/error-invalid-path-during-git-clone-to-windows-client-1085186345.html

Reply:
👍 No to jsem nevěděl, ale nepřekvapuje mě to. 😃